qlog/src/events/quic.rs

```diff
@@ -494,6 +497,33 @@ pub enum QuicFrame {
         raw: Option<Bytes>,
     },
 
+    AckMp {
+        space_identifier: u64,
+
+        ack_delay: Option<f32>,
+        acked_ranges: Option<AckedRanges>,
+
+        ect1: Option<u64>,
+
+        ect0: Option<u64>,
+
+        ce: Option<u64>,
+    },
+
+    PathAbandon {
+        identifier_type: u8,
```

Comment:
in the absence of a qlog spec, I'd expect `identifier_type` to be u64. That makes them consistent with varint type in the frame definition.